lib/api/client-commands/cookies/getAll.js

```diff
@@ -0,0 +1,48 @@
+const ClientCommand = require('../_base-command.js');
+const {Logger} = require('../../../utils');
+
+/**
+ * Retrieve all cookies visible to the current page.
+ *
+ * The cookies are returned as an array of cookie JSON object, with properties as defined [here](https://www.w3.org/TR/webdriver/#dfn-table-for-cookie-conversion).
+ *
+ * @example
+ * module.exports = {
+ *   'get all cookies': function (browser) {
+ *     browser
+ *       .cookies.getAll(function (result) {
+ *         this.assert.equal(result.value.length, 1);
+ *         this.assert.equal(result.value[0].name, 'test_cookie');
+ *       });
+ *   },
+ *
+ *   'get all cookies with ES6 async/await': async function (browser) {
+ *     const cookies = await browser.cookies.getAll();
+ *     browser.assert.equal(cookies.length, 1);
+ *     browser.assert.equal(cookies[0].name, 'test_cookie');
+ *   }
+ * };
+ *
+ * @syntax .cookies.getAll([callback])
+ * @method cookies.getAll
+ * @param {function} [callback] Callback function which will receive the response as an argument.
+ * @returns {Array.<object>} A list of cookie JSON objects, with properties as defined [here](https://www.w3.org/TR/webdriver/#dfn-table-for-cookie-conversion).
+ * @see cookies.get
+ * @see cookies.set
+ * @see cookies.delete
+ * @see cookies.deleteAll
+ * @api protocol.cookies
+ */
+
+class GetCookies extends ClientCommand {
+  performAction(callback) {
+    this.transportActions
+      .getCookies(callback)
+      .catch(err => {
+        Logger.error(err);
```

Comment:
do we need the logs here?

Reply:
Yes, because for certain errors, the error is not reported in the terminal.

To reproduce, go to `deleteCookie` method in method-mappings and change `manage().deleteCookie(cookieName)` to `manage().deleteCookies(cookieName)`. Now, if you run the `cookies.deleteCookie()` command, you would see a TypeError in terminal, but if you remove the `Logger.error(err)` statement above you won't.

More context: https://github.com/nightwatchjs/nightwatch/pull/3208#discussion_r894412900

Reply:
got it